README.md

```diff
@@ -9,13 +9,13 @@ The preferred way to install this extension is through [composer](http://getcomp
 Either run:
 
 ```
-php composer.phar require --prefer-dist yetopen/yii2-usuario-ldap "*"
+php composer.phar require --prefer-dist yetopen/yii2-usuario-ldap-extension "*"
```

Comment:
I don't understand this change, in `composer.json` it is still named `yetopen/yii2-usuario-ldap`

Reply:
Sorry, this is a leftover. Just remove it.